chef/cookbooks/glance/templates/default/glance-api.conf.erb

```diff
@@ -483,7 +484,7 @@ filesystem_store_datadir = <%= node[:glance][:filesystem_store_datadir] %>
 
 # Version of the authentication service to use
 # Valid versions are '2' for keystone and '1' for swauth and rackspace
-swift_store_auth_version = 2
+swift_store_auth_version = <%= @keystone_settings['api_version'] %>
```

Comment:
Are you sure this is the API version? The comment above the line seems to say something else.